articles/00-GettingStarted.md

```diff
@@ -66,32 +84,94 @@ Which contains the following:
+
+### Building for Windows Mixed Reality (UWP)
+
+Unity allows running of UWP projects directly from the editor for simple testing.  Builds that are exported will require Visual Studio with the latest UWP API's installed to produce a build to ship and run on other WIndows 10 machines or the HoloLens.  Visual Studio also provides an independent method to publish to the Windows Store for distribution.
+
+Please ensure the `Windows Mixed Reality` XRTK package is installed and that the Windows Mixed Reality Camera and controller providers are configured.
+
+### Building for Oculus Quest / Android
+
+Building for Oculus Quest is rather unique, if you have your Oculus Quest connected to your machine via a high speed USB3 cable and have the Oculus Client running, you can run Quest project in Windows Standalone mode directly on the Quest via Oculus Link, [please see Oculus Support for further details](https://support.oculus.com/444256562873335/).  This will enable both controller and hands support for quick development.
+
+When building for Android, there is an additional requirement to install an `AndroidManaifest` for the quest to operate correctly, the XRTK provides a fast an easy way to do this by simply clicking `Mixed Reality Toolkit -> Tools -> Oculus -> Create Oculus Quest Compatible AndroidManaifest.xml` in the Unity editor menu. (**without this, your project may not start or appear in 2D**)
+
+Please ensure the `Oculus` XRTK package is installed and that the Oculus Camera and controller providers are configured.
+
+### Building for Magic Leap
+
+Magic Leap is the one exception that does require the new Unity XR Management system and will require a separate Unity project at this time, primarily due the way the Magic leap platform is evolving.  We recommend building your project for one of the other platforms and then simply copying the asset folder to a new Unity project for Magic Leap to build and run.
+
+XRTK handles all the complexities underneath, so there will be no difference between your Magic Leap project and other platforms.
+
+> As soon as the new Unity XR Management system is fully supported by the XRTK, this separate project will not be required.
+
+To get started, ensure that the following components are installed:
 
-![Build Window](../images/GettingStarted/MixedRealityBuildWindow.png)
+* Unity Magic Leap XR package installed
+* Magic Leap Unity asset installed (from Magic Leap hub)
+* Magic Leap Developer Hub
+* Visual Studio 2019
 
-Once the build completes, you can install and run it on the device.
+Please ensure the `Lumin` XRTK package is installed and that the Lumin Camera and controller providers are configured.
 
-> **Note:** The build window automatically installs some platforms for you, but depending on the platform, you'll need to follow the platform specific installation steps to get it manually installed on your device.
+**A more detailed guide for building for Magic leap will follow soon**
```

Comment:
I think this article already gets too much into platform specific setup requirements. It might be better to break out each platform specific setup into their own articles and keep this one as generic as possible with links to platform specific articles as needed.

Reply:
I have created individual Platform documents with specifics and a lot more highlights.  These are just the short versions for the getting started guide.

But as I stated, I needed assistance to update the Magic Leap version